src/components/modals/details-qr.tsx

```diff
@@ -28,80 +28,83 @@ export interface DetailsQrModalProps {
   address: string;
 }
 
-export const DetailsQrModal = ({address, onClose}: Modals['cardDetailsQr']) => {
-  const svg = useRef();
-  const wallet = useWallet(address) as Wallet;
-  const {width} = useWindowDimensions();
+export const DetailsQrModal = observer(
+  ({address, onClose}: Modals['cardDetailsQr']) => {
+    const svg = useRef();
+    // TODO wallet can be null
+    const wallet = Wallet.getById(address) as Wallet;
```

Comment:
According to previous comment
```suggestion
    const wallet = Wallet.getById(address) as (Wallet | null);
```

Reply:
getById returns Wallet or null. We don't need this assertion here at all. If I remove  "as Wallet" we get errors down by tree because children expect Wallet but getById can return null. I don't know we it implemented like this but looks like it is error. I left TODOs and will fix it in separate PR.